src/schemas/raster.schema.json

```diff
@@ -43,22 +43,58 @@
     "metadata": {
       "type": "object",
       "additionalProperties": false,
-      "required": ["dimensions", "is_pyramid", "transform"],
+      "required": [],
```

Comment:
This worries me a bit... I'm not sure how much variation you need to handle, but another approach would be to define different valid subsets of required properties... or to have different elements, instead of a one-size fits all metadata.

Reply:
The 3 cases are basically `["dimensions", "is_pyramid", "transform"]`, `["omeTiffOffsetsUrl"]`, or nothing at all (i.e no `metadata`).  I am not sure there is a way to do this in JSON schema so the best option may be to make `metadata` optional, leave its current 3 checks, and then move `omeTiffOffsetsUrl` out into its own field.

Reply:
@ilan-gold : you could use [`oneOf`](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof) ... but I think you're right that `omeTiffOffsetsUrl` could be moved out... unless you don't want it to be too OME specific at a high level?

The downside of the `oneOf` is that validation failures become more cryptic... if it doesn't validate, it's not sure which one you intended.

Use your judgement. Thanks!